lib/wal/sources/Wal.cpp

```diff
@@ -0,0 +1,67 @@
+//
+// Created by Zoe Roux on 2021-05-14.
+//
+
+#include <chrono>
+#include "Wal.hpp"
+
+using namespace std::chrono_literals;
+
+namespace WAL
+{
+	std::chrono::nanoseconds Wal::timestep = 8ms;
+
+	SceneManager &Wal::getSceneManger()
```

Comment:
MANGER !!!
![cc you reading this](https://o3.llb.be/image/thumb/5617d5763570b0f19f3eaf68.jpg)

Reply:
Oups, I feel called out